### DISCUSSION TITLE
Facing issues on Git via terminal of Macbook M1

### DISCUSSION THREAD

**Discussion by rakeshnaidug99** [community/community#120838] [New to GitHub]
I am new to Git and trying to push my first repo, of my devops-course, and i am getting errors while creating simple 1.txt (comitted) 2.txt  (staged and modified)and 3.txt(created and unstaged ) files, I have been facing issues and this is what my terminal looks like, i am also having issues to authenticate via terminal, as even if i type the correct credentials it says, authentication failed, i tried to change directories but still failed, can some one help?

or Are there any tutorials for Git hub via mac zsh terminal???

**Comment by gurjeetsinghvirdee** [comment]
Hey @rakeshnaidug99 
You can go through this tutorial: https://www.youtube.com/watch?v=p0Js7IF17yI
